webpack.config.js

```diff
@@ -85,13 +85,13 @@ module.exports = {
     quiet: false,
     disableHostCheck: true,
     host: '0.0.0.0',
-    port: 3000,
+    port: `${process.env.FRONTEND_URL}`.replace(/\D/g, ''),
```

Comment:
Maybe `Url` is available here to do this without string replace? https://developer.mozilla.org/en-US/docs/Web/API/URL/port

Reply:
Did this! Went with your suggestion for using `const frontendUrl = process.env.FRONTEND_URL || 'http://localhost:3000'` to avoid circleCI failure because of `new URL(...)` throwing an error when it builds in git since .env isn't available